tools/travis-test.sh

```diff
@@ -142,9 +123,7 @@ run_tests() {
   echo "Running big tests (tests/ejabberd_tests)"
   echo "############################"
 
-  for node in ${NODES[@]}; do
-    start_node $node;
-  done
+  time ${TOOLS}/start-nodes.sh
 
   # Start all additional services
   start_services
```

Comment:
Would it make sense to also reorder instructions here to `start_services` before `start-nodes.sh`?

Reply:
Ok, I think that's out of scope here.